src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -108,6 +108,7 @@ import { DateUtilities } from '@/services'
 })
 export default class AgmExtensionEvaluation extends Vue {
   @Prop({ required: true }) readonly data!: AgmExtEvalIF
+  @Prop({ default: false }) readonly evaluateResult!: boolean
```

Comment:
This is specific enough, and not part of "data", that I'm OK with this being a new prop 👍